be/src/util/mysql_row_buffer.cpp

```diff
@@ -44,6 +44,9 @@ namespace doris {
 static uint8_t NEXT_TWO_BYTE = 252;
 static uint8_t NEXT_THREE_BYTE = 253;
 static uint8_t NEXT_EIGHT_BYTE = 254;
+// the EXTRA_RESERVE_BYTE wanner to make sure _pos pointer is always in _buf memory
+// used in reserve() for allocate current buffer
+static size_t EXTRA_RESERVE_BYTE = 128;
```

Comment:
Why choose 128 bytes? Will it cause too much overhead for each row?

Reply:
commit add to line 139

Reply:
maybe 16 is enough

Reply:
change 128 to 16